release_notes/371-release-notes.adoc

```diff
@@ -0,0 +1,225 @@
+:_content-type: ASSEMBLY
+[id="release-notes-371"]
+= Red Hat Advanced Cluster Security for Kubernetes 3.71
+include::modules/common-attributes.adoc[]
+:context: release-notes-371
+
+toc::[]
+
+*Release date:* 25 July 2022
+
+{product-title} ({product-title-short}) is an enterprise-ready, Kubernetes-native container security solution that protects your vital applications across build, deploy, and runtime. It deploys in your infrastructure and integrates with your DevOps tooling and workflows to deliver better security and compliance and to enable DevOps and InfoSec teams to operationalize security.
```

Comment:
I have a red flag for "tooling": 

Do not use unless the term is part of the existing product terminology. Use a simpler term such as "tool" or "tools".

Maybe it *is* part of the existing product terminology? Just bringing it to your attention.

Reply:
That is marketing's language - I'll ask.